src/fundus/publishers/es/la_vanguardia.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+from typing import List, Optional
+
+from lxml.etree import XPath
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class LaVanguardiaParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = XPath("//div[@class='article-modules']//p[@class='paragraph']")
+        _subheadline_selector = XPath("//div[@class='article-modules']//h3[@class='subtitle']")
+        _summary_selector = XPath("//h2[@class='epigraph']")
+
+        @attribute
+        def body(self) -> Optional[ArticleBody]:
```

Comment:
The selector seems to have trouble parsing [this](https://www.lavanguardia.com/internacional/20241005/9997061/gaza-aniversario-inicio-guerra-siete-octubre.html) article

Reply:
It seems to me, as if there is nothing we can do about it, since the content is loaded using a script. The HTML we get in Fundus seems to mostly be scripts